planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -1265,8 +1268,12 @@ TurnSignalInfo StartPlannerModule::calcTurnSignalInfo()
   // In Geometric pull out, the ego stops once and then steers the wheels to the opposite direction.
   // This sometimes causes the getBehaviorTurnSignalInfo method to detect the ego as stopped and
   // close to complete its shift, so it wrongly turns off the blinkers, this override helps avoid
-  // this issue.
+  // this issue. Also, if the ego is not engaged (so it is stopped), the blinkers should still be
+  // activated.
   const bool override_ego_stopped_check = std::invoke([&]() {
+    if (!status_.first_engaged_and_driving_forward_time) {
```

Comment:
and change this condition to `needToPrepareBlinkerBeforeStratDrivingForward()` ?

Reply:
@soblin I cannot check the current speed, I need a check that tells me if at some point in the PAST the ego has moved faster than the threshold speed. So this approach cannot work.